ironfish/src/rpc/routes/wallet/getBalances.ts

```diff
@@ -86,6 +101,7 @@ routes.register<typeof GetBalancesRequestSchema, GetBalancesResponse>(
       const asset = await account.getAsset(balance.assetId)
 
       balances.push({
+        asset: asset ? constructRpcAsset(asset) : undefined,
         assetId: balance.assetId.toString('hex'),
         assetName: asset?.name.toString('hex') ?? '',
         assetCreator: asset?.creator.toString('hex') ?? '',
```

Comment:
Look for part of the code where transactions and assets get added to the wallet db